src/status_im/subs/profile.cljs

```diff
@@ -308,3 +308,9 @@
  :<- [:profile/profile]
  (fn [profile]
    (not (boolean (seq (:mnemonic profile))))))
+
+(re-frame/reg-sub
+ :profile/logged-in?
```

Comment:
We already have `:multiaccount/logged-in?`. Unrelated but implementation of `:multiaccount/logged-in?` seems wrong, if possible please move this code there.

Reply:
Thank you, if possible please update code of `:multiaccount/logged-in?` with your implemation. It subscribes to whole db.

```
(re-frame/reg-sub
 :multiaccount/logged-in?
 (fn [db]
   (multiaccounts.model/logged-in? db)))
```